### PR TITLE
Feature 1

### PR DESCRIPTION
**Pull request by amiralda**
This is the initial version of my hello microverse project.

### CODE REVIEW

**Review by shella12 — CHANGES_REQUESTED** [review]
Hi @amiralda , 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights  

- Correct use of git flow ✅  

## Required Changes ♻️

- [x] _Check the comments under the review._
- [x]  While you made a great  👍  effort adding linter files 📁 to your project, you still have to fix your linter errors . Please fix your `stylelint` errors. You can do this manually or you can use auto fix command in your terminal (if you have installed linter checkers locally if not please use [these guideline](https://github.com/microverseinc/linters-config/blob/master/html-css/README.md) to install them locally):
```sh
    npx stylelint "**/*.{css,scss}" --fix  
```
- [ ] [OPTIONAL] Please use descriptive commit messages. The commit message should be written in the imperative tense and capitalized. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8). Its one of the requirements of the project but please try to keep this in mind for future

Screenshot 2023-02-16 at 8 23 42 AM

- [ ] I suggest 🤓  using a more descriptive name for your feature branch instead of `feature-1`. Branch should have a name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section’) 😄 

Screenshot 2023-02-16 at 8 12 39 AM


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,259 @@
...
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
```
[the remark this answers is not in the document]
Comment (amiralda) [reply README.md:49]: Done

File: README.md [hunk 2]
```diff
@@ -1 +1,259 @@
...
+    <li><a href="https://expressjs.com/">N/A</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">N/A</a></li>
+  </ul>
+</details>
```
[the remark this answers is not in the document]
Comment (amiralda) [reply README.md:85]: Done

File: .gitignore [hunk 3]
```diff
@@ -1,4 +1,6 @@
+test.md 
 # Logs
+node_modules/
 logs
 *.log
 npm-debug.log*
```
[the remark this answers is not in the document]
Comment (amiralda) [reply .gitignore:6]: Thank you!

File: README.md [hunk 4]
```diff
@@ -1 +1,259 @@
...
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
[the remark this answers is not in the document]
Comment (amiralda) [reply README.md]: Done

**Pull request closed without merging**